lib/profiles/dosomething/drupal-org.make

```diff
@@ -253,6 +253,10 @@ projects[stage_file_proxy][subdir] = "contrib"
 ;libraries[dsapi][download][type] = "git"
 ;libraries[dsapi][download][url] = "git@github.com:DoSomething/dsapi-php.git"
 
+; Guzzle
+libraries[guzzle][download][type] = "git"
+libraries[guzzle][download][url] = "https://github.com/guzzle/guzzle.git"
```

Comment:
conflicted on that, I don't want to get super behind on the version, as they release new stuff frequently. Maybe these transactions is something we could add monitoring into?